src/mlte/evidence/evidence_metadata.py

```diff
@@ -0,0 +1,60 @@
+"""
+EvidenceMetadata class definition.
```

Comment:
I like this name much better than `MeasurementMetadata`. Thank you for coming up with something good!